command/commands/user.py

```diff
@@ -28,7 +28,7 @@ class UserCommand:
     """Parser for view command."""
     parser_view = subparsers.add_parser("view")
     parser_view.set_defaults(which="view")
-    parser_view.add_argument("slack_id", type=str, action='store')
+    parser_view.add_argument("--slack_id", type=str, action='store')
```

Comment:
Since the Slack ID is required to use the `view` command, this probably shouldn't be a full flag but rather a required argument (though I'm not sure off the top of my head how to do that); that is, per the docs, the command should be `@rocket user view SLACK_ID`, but this will parse `@rocket user view --slack_id SLACK_ID`, which is slightly less convenient.

Reply:
I made it optional so that if the argument isn't included, then it'll just view the information of the user who called the command. The optional `--slack_id` is for if you want to view information about another user's info